websites/A/Audiomack/dist/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "Runa ٩(＾◡＾)۶",
```

Comment:
Is it required to use our Discord name here? As I primarily go by Kanna and only using Runa as a temporary name.

Reply:
Feel free to use Kanna

Reply:
Alright 👍🏻